src/nbsap/management/commands/import_nat.py

```diff
@@ -60,13 +60,26 @@ def handle(self, *args, **options):
                     row = row[1:]
                 data.append(row)
 
+        imported_titles = []
         for row in data:
-            nat_obj = NationalObjective.objects.create()
-            setattr(nat_obj, 'title_' + language, row[0])
-            setattr(nat_obj, 'description_' + language, row[1])
-            nat_obj.save()
-            strategy = NationalStrategy.objects.create(objective=nat_obj)
-            eu_targets = [c for c in row[2].split(',') if c]
+            obj_title = row[0]
+            if obj_title not in imported_titles:
+                # import objective
+                nat_obj = NationalObjective()
+                setattr(nat_obj, 'title_' + language, obj_title)
+                setattr(nat_obj, 'description_' + language, row[2])
+                nat_obj.save()
+                self.stdout.write(u'Imported objective {0}'.format(nat_obj))
+                imported_titles.append(obj_title)
+            # import subobjective
+            nat_sub_obj = NationalObjective()
+            setattr(nat_sub_obj, 'title_' + language, row[1])
+            setattr(nat_sub_obj, 'description_' +  language, row[3])
+            nat_sub_obj.parent = nat_obj
+            nat_sub_obj.save()
```

Comment:
What if this a single subobjective? A NationalObjective with a single child will be created.

How does this work for Hungary, where column C (national_target_main_ref_level2) is always empty?

I would suggest rewriting the flow as a two step process:
- step 1: parse file, create a data structure of Objectives/Subobjectives properties
- step 2: save the data structure into the database, taking care of special cases such as single child